workflow-gen/Activities/ProcessPaymentActivity.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Threading.Tasks;
+using Dapr.Client;
+using Dapr.Workflow;
+using Microsoft.Extensions.Logging;
+using WorkflowGen.Models;
+using System;
+using Microsoft.Extensions.Configuration;
+
+namespace WorkflowGen.Activities
+{
+
+    class ProcessPaymentActivity : WorkflowActivity<PaymentRequest, object>
+    {
+        readonly ILogger logger;
+        readonly DaprClient client;
+        IConfiguration appSettings;
+
+        public ProcessPaymentActivity(ILoggerFactory loggerFactory, DaprClient client)
+        {
+            this.logger = loggerFactory.CreateLogger<ProcessPaymentActivity>();
+            this.client = client;
+            this.appSettings = appSettings = new ConfigurationBuilder()
```

Comment:
Why the `=` in the middle? Also, ideally, we would've setup this config in startup or program and injected it as an `IConfiguration`.

Reply:
I am not quite sure how the IConfiguration injection works currently.